code/modules/cm_marines/Xeno_Research/Xeno_Analyzation.dm

```diff
@@ -0,0 +1,257 @@
+//// Research Stuff ////
+/obj/machinery/alien_container
+	name = "Speciemen Analyze Chamber"
+	desc = "Standart pressure chamber for all your biology needs"
+	icon = 'icons/Marine/Research/Research_Machinery.dmi'
+	icon_state = "analyzer_chamber"
+	var/mob/living/carbon/Xenomorph/occupant = null
+	var/obj/machinery/computer/analyze_console/linked_console = null
+
+	use_power = 1
+	idle_power_usage = 20
+	active_power_usage = 200
+
+/obj/machinery/alien_container/update_icon()
+	if(occupant)
+		icon_state = "analyzer_chamber_full"
+		return
+	icon_state = "analyzer_chamber"
+
+/obj/machinery/alien_container/attackby(obj/item/W, mob/living/user)
+	if(istype(W, /obj/item/grab))
+		var/obj/item/grab/G = W
+		if(!ismob(G.grabbed_thing))
+			return
+		var/mob/M = G.grabbed_thing
+		if(isXeno(M))
+			put_mob(M)
+	return
+
+/obj/machinery/alien_container/proc/put_mob(mob/living/carbon/M as mob)
+	if (stat & (NOPOWER|BROKEN))
+		to_chat(usr, "\red The chamber is not functioning.")
+		return
+	if (!istype(M))
+		to_chat(usr, "\red <B>The chamber cannot handle such a lifeform!</B>")
+		return
+	if (!isXeno(M))
+		to_chat(usr, "\red <B>You feel stupid...</B>")
```

Comment:
You can simply remove this altogheter and change the proc's argument var to `mob/living/carbon/Xenomorph/M`.